### PR TITLE
Add CONTEXT param to buildah task.

### PR DESCRIPTION
**Pull request by gmarcy**
# Changes

Add CONTEXT param to buildah task.  Addresses issue #170.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x ] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by gmarcy** [comment]
@googlebot I signed it!

**Comment by csantanapr** [comment]
I hit the same problem and after debugging a bit with @gmarcy , we fixed using this approach.
We have many projects that `Dockerfile` will not be in the root of the git repository

File: buildah/README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/build
   note below.  (_default:_ quay.io/buildah/stable:v1.11.0)
 * **DOCKERFILE**: The path to the `Dockerfile` to execute (_default:_
   `./Dockerfile`)
+* **CONTEXT**: Path to the directory to use as context (_default:_
```
Comment (csantanapr) [inline buildah/README.md]: Using the name `CONTEXT` brings parity with other builder tools like kaniko, easy to swap tasks if they have the same input names.
Ref: https://github.com/tektoncd/catalog/blob/master/kaniko/kaniko.yaml#L11

**Comment by csantanapr** [comment]
@vdemeester 

Does OpenShift use this file as the canonical source for clustertask in OpenShift Pipeline Operator catalog?
No need to submit the patch to another repo?

I'm guessing when a new release get's cut for tektoncd then openshift will pick via a new branch here https://github.com/openshift/tektoncd-catalog

**Comment by vdemeester** [comment]
@csantanapr yeah it should be picked up by our nightly and will be in the 0.10.0 release of the OpenShift Pipeline Operator :wink:

**Pull request merged**